src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, memo, ReactNode } from 'react';
+import { Tab as ReactTab, TabList, TabPanel, Tabs as ReactTabs } from 'react-tabs';
+
+import { Button } from '@material-ui/core';
+
+import useStyles from './styles';
+
+interface Tab {
+  label: string;
+  component: ReactNode;
+}
+
+interface TabsProp {
+  tabs: Tab[];
+}
+
+const Tabs: FC<TabsProp> = ({ tabs }) => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <ReactTabs>
+        <TabList>
+          {tabs.map(tab => (
+            <ReactTab>
```

Comment:
Need to add a key here (I missed this earlier).
```js
{tabs.map((tab, index) => (
  <ReactTab key={index}>
    ...
```

Reply:
Using the index is not recommended so I used `shortid.generate()` to create a key for these components